docs/order_of_PCI_addresses.md

```diff
@@ -0,0 +1,28 @@
+Order of PCI addresses
+===============================
+
+## Standard PCI addressing
+
+Coreboots standard way of numbering PCI devices prioritizes PCIe slots over
+embedded PCI devices like Ethernet interfaces. It causes weird and unwanted
+behavior like shifting addresses of old devices when a new one is added.
+Unfortunately it can’t be changed in APU without consequences, because of many
+years of it existence on the market. If it had been changed, many APU owners
+would be surprised by e.g. change of interfaces names.
+
+## sortbootorder
+
+To address this issue, we added a runtime option in [sortbootorder](https://github.com/pcengines/sortbootorder/blob/master/README.md)
+which can reverse those priorities. By setting `Reverse order of PCI addresses`
```

Comment:
I would not use priorities, because that makes no sense. Write something like this:
```
Reverses the order of PCIe engines assignment in a way that:
- NICs occupy the lowest PCIe device numbers which make their enumeration persistent regardless of the WiFi module presence
- the WoL capable NIC is the first PCIe device in iPXE and in the operating system
- the mPCIe slots occupy the highest PCIe device numbers in following order: 3x NIC, mPCIe1, mPCIe2. In such case faulty detected WiFi in mPCIe2 slot will not affect interface renaming for mPCIe1 slot WiFi card.

By default, this option is disabled which means nothing changes (compared to the state maintained for past 4 years) without intentional change of this setting.
```